taskgraph/Task.py

```diff
@@ -1478,7 +1580,9 @@ def _normalize_path(path):
     return os.path.normcase(abs_path)
 
 
-@retrying.retry(wait_exponential_multiplier=1000, wait_exponential_max=5000)
+@retrying.retry(
+    wait_exponential_multiplier=100, wait_exponential_max=3200,
+    stop_max_attempt_number=5)

```

Comment:
Cool, `wait_exponential_multiplier` is a great idea I never knew existed.

Reply:
Yep, I use it all the time when doing internet stuff. It comes from the classic TCP congestion control algorithm if you're interested: https://en.wikipedia.org/wiki/TCP_congestion_control. It leads to one of my favorite computer science terms "congestion collapse".